pennylane_lightning/src/tests/CMakeLists.txt

```diff
@@ -32,13 +32,14 @@ include(Catch)
 add_executable(runner runner_main.cpp)
 target_link_libraries(runner lightning_simulator lightning_utils lightning_algorithms Catch2::Catch2)
 
-target_sources(runner PRIVATE   Test_AdjDiff.cpp
-                                Test_Bindings.cpp
+target_sources(runner PRIVATE   Test_Util.cpp
+                                Test_AdjDiff.cpp
+                                Test_VectorJacobianProduct.cpp
                                 Test_StateVector_Nonparam.cpp 
                                 Test_StateVector_Param.cpp 
                                 Test_StateVectorManaged_Nonparam.cpp 
                                 Test_StateVectorManaged_Param.cpp 
-                                Test_Util.cpp
+                                Test_Bindings.cpp
```

Comment:
Just curious about the reordering of these? If ordering doesn't matter, I think keeping alphabetical may be a good idea.

Reply:
Nice catch! It wasn't intentional :slightly_smiling_face: